clients/geth/specular/data/genesis.json

```diff
@@ -14,8 +14,22 @@
+      "code": "0x6080604052600436106100745760003560e01c80638da5cb5b1161004e5780638da5cb5b146100f75780639ad2cd1414610124578063a4beda6314610148578063f2fde38b1461017557600080fd5b80632e64cec1146100b8578063544c7cf9146100cf578063715018a6146100e257600080fd5b366100b357604080513381523460208201527f3916187166447a01764df803fb1147a37c55a4ca1c7271a7a75c0ab1079b2d95910160405180910390a1005b600080fd5b3480156100c457600080fd5b506100cd610195565b005b6100cd6100dd366004610467565b6101cc565b3480156100ee57600080fd5b506100cd610333565b34801561010357600080fd5b506000546040516001600160a01b0390911681526020015b60405180910390f35b34801561013057600080fd5b5061013a60015481565b60405190815260200161011b565b34801561015457600080fd5b5061013a610163366004610467565b60026020526000908152604090205481565b34801561018157600080fd5b506100cd610190366004610467565b610347565b61019d6103bd565b60405133904780156108fc02916000818181858888f193505050501580156101c9573d6000803e3d6000fd5b50565b6101d46103bd565b6001600160a01b03811660009081526002602052604090205442116102405760405162461bcd60e51b815260206004820152601a60248201527f4c6f636b2074696d6520686173206e6f7420657870697265642e00000000000060448201526064015b60405180910390fd5b60015447116102915760405162461bcd60e51b815260206004820152601b60248201527f4e6f7420656e6f7567682066756e647320696e206661756365742e00000000006044820152606401610237565b6001546040516001600160a01b0383169180156108fc02916000818181858888f193505050501580156102c8573d6000803e3d6000fd5b506102d6426201518061048a565b6001600160a01b038216600081815260026020908152604091829020939093556001548151928352928201929092527fc3b176023218681566723c842ee1b4f61fb302c5af042b16c05d11d68dedffd8910160405180910390a150565b61033b6103bd565b6103456000610417565b565b61034f6103bd565b6001600160a01b0381166103b45760405162461bcd60e51b815260206004820152602660248201527f4f776e61626c653a206e6577206f776e657220697320746865207a65726f206160448201526564647265737360d01b6064820152608401610237565b6101c981610417565b6000546001600160a01b031633146103455760405162461bcd60e51b815260206004820181905260248201527f4f776e61626c653a2063616c6c6572206973206e6f7420746865206f776e65726044820152606401610237565b600080546001600160a01b038381166001600160a01b0319831681178455604051919092169283917f8be0079c531659141344cd1fd0a4f28419497f9722a3daafe3b4186f6b6457e09190a35050565b600060208284031215610478578081fd5b8135610483816104ae565b9392505050565b600082198211156104a957634e487b7160e01b81526011600452602481fd5b500190565b6001600160a01b03811681146101c957600080fdfea2646970667358221220ac7e90a71250a47e2f706c29c99ee99f591fa5d45d916975565631f57e1870aa64736f6c63430008040033",
+      "balance": "100000000000000000000",
+      "storage": {
+        "0x0000000000000000000000000000000000000000000000000000000000000000": "0x000000000000000000000000f39fd6e51aad88f6f4ce6ab8827279cfffb92266",
+        "0x0000000000000000000000000000000000000000000000000000000000000001": "0x0000000000000000000000000000000000000000000000000de0b6b3a7640000"
+      }
+    }
   }
-}
+}
```

Comment:
don't check in these changes—otherwise it'll have to be kept in sync w/ calling `create_genesis`